.github/workflows/main_ci.yml

```diff
@@ -196,7 +215,7 @@ jobs:
     runs-on: macos-latest
 
     env:
-        CMAKE_BUILD_DIR: ${{ github.workspace }}/build
+        BUILD_DIR: ${{ github.workspace }}/build
         VCPKG_BINARY_SOURCES: files,${{ github.workspace }}/build/cache,readwrite
```

Comment:
I know you did not change anything, but this is redundant. It should be at the top.